app/root.tsx

```diff
@@ -119,6 +120,13 @@ export async function loader({ request }: LoaderFunctionArgs) {
   const recaptchaSiteKey = getEnvOrDieInProduction('RECAPTCHA_SITE_KEY')
   const userIsMod = user?.groups.includes(moderatorGroup)
   const userIsVerifiedSubmitter = user?.groups.includes(submitterGroup)
+  let publicConsumerCredential
+  if (user) {
+    const machine = await ClientCredentialVendingMachine.create(request)
+    publicConsumerCredential = (await machine.getClientCredentials())?.find(
+      (x) => x.scope == 'gcn.nasa.gov/kafka-public-consumer'
+    )?.client_id
+  }
```

Comment:
Oof. This will result in Cognito API calls for loading any page at all. I don't think that this is acceptable from a performance perspective.

Reply:
What are your thoughts on how to fix this? We could add a route that is just a loader and redirect?

Reply:
I think that you should not automatically select the client ID.

Reply:
Addressed in latest commit